.clang-format

```diff
@@ -1,3 +1,7 @@
-Language:        JavaScript
-BasedOnStyle:    Google
-ColumnLimit:     100
+Language: JavaScript
+BasedOnStyle: Google
+ColumnLimit: 100
+---
+Language: Proto
+# Don't format .proto files.
```

Comment:
that's cool, but why not? I think they are all third_party - maybe we meant to exclude third_party directory?

Reply:
Sure. Separate PR?